packages/main/src/TableGrowing.ts

```diff
@@ -144,6 +144,7 @@ class TableGrowing extends UI5Element implements ITableGrowing {
 	onTableActivate(table: Table): void {
 		this._table = table;
 		this._shouldFocusRow = false;
+		this._invalidateTable();
```

Comment:
this only needed for the Scroll mode right? 

BTW shouldn't this be better handled after first rendering of the table after shadow dom of the table is ready?

A test would be nice.

Reply:
You're right. Moving this to onTableAfterRendering, as rerendering is only necessary if after rendering the `this._table` property is not initialized yet.
Test for this particular case already exists in TableGrowing.cy.tsx (see "tests button shown when not scrollable"). This is why, the PR mentioned above is failing